src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -7,6 +7,7 @@
 	<changes>
 	<![CDATA[
 	Update add-on's info URL.<br>
+	Added new scanner TestUserAgent. The scanner checks for differences in response based on fuzzed User Agent.<br>

```

Comment:
Consider using "User Agent Fuzzer" instead of "TestUserAgent", the former is the name shown to the user.